chore: Update README (#1402)
tensorchord/envd: README.md

<a href='https://coveralls.io/github/tensorchord/envd?branch=main'><img src='https://coveralls.io/repos/github/tensorchord/envd/badge.svg?branch=main' alt='Coverage Status' /></a>
</p>

## What is envd?

envd (`ɪnˈvdɪ`) is a command-line tool that helps you create the container-based development environment for AI/ML.

> Creating development environments is no easy task, especially with today's complex systems and dependencies. With everything from Python to CUDA, BASH scripts and Dockerfiles constantly breaking it can feel like a nightmare - until now!

Reviewer: ```suggestion
Creating development environments is not easy, especially with today's complex systems and dependencies. With everything from Python to CUDA, BASH scripts, and Dockerfiles constantly breaking, it can feel like a nightmare - until now!
```

Environments built with `envd` provide the following features out-of-the-box:

> **Simple CLI, and language**

Reviewer: ```suggestion
**Simple CLI and language**
```

> `envd` enables you to quickly and seamlessly integrate powerful CLI tools into your existing Python workflow - no extra learning required! Provision your programming environment with envd's simple language.

Reviewer: ```suggestion
`envd` enables you to quickly and seamlessly integrate powerful CLI tools into your existing Python workflow to provision your programming environment without learning a new language or DSL.
```

```python
def build():
    install.python_packages(name = [
        "numpy",
    ])
    shell("zsh")
    config.jupyter()
```

**Isolation, compatible with OCI image**

> With `envd`, users can create a isolated space to train, fine-tune or serving. By utilizing sophisticated virtualization technology as well as other features like [buildkit](https://github.com/moby/buildkit), it's an ideal solution for environment setup.

Reviewer: ```suggestion
With `envd`, users can create an isolated space to train, fine-tune, or serve. By utilizing sophisticated virtualization technology as well as other features like [buildkit](https://github.com/moby/buildkit), it's an ideal solution for environment setup.
```

`envd` environment image is compatible with [OCI image specification](https://github.com/opencontainers/image-spec). By leveraging the power of an OCI image, you can make your environment available to anyone and everyone! Make it happen with a container registry like Harbor or Docker Hub.

**Local, and cloud**

> `envd` can now be used on a hybrid platform, ranging from the local machines to clusters hosted by Kubernetes. Any of these options offer an efficient and versatile way for developers to create their projects!

Reviewer 2: ```suggestion
`envd` can now be used on a hybrid platform, ranging from local machines to clusters hosted by Kubernetes. Any of these options offers an efficient and versatile way for developers to create their projects!
```

```sh
$ envd context use local
# Run envd environments locally
$ envd up
...
$ envd context use cluster
# Run envd environments in the cluster with the same experience
$ envd up
```

Check out the [doc](https://envd.tensorchord.ai/teams/kubernetes.html) for more details.

**Build anywhere, faster**

`envd` offers a wealth of advantages, such as remote build and software caching capabilities like pip index caches or apt cache, with the help of [buildkit](https://github.com/moby/buildkit) - all designed to make your life easier without ever having to step foot in the code itself!

Reusing previously downloaded packages from the PyPI/APT cache saves time and energy, making builds more efficient. No need to redownload what was already acquired before – a single download is enough for repeat usage! 

> With dockerfile v1, users are unable to take advantage of PyPI caching for faster installation speeds - but `envd` offers this support and more!

Reviewer 2: ```suggestion
With Dockerfile v1, users are unable to take advantage of PyPI caching for faster installation speeds - but `envd` offers this support and more!
```

<p align=center>
  <img src="https://user-images.githubusercontent.com/5100735/189928628-543f4851-87b7-462b-b811-372cbf46ff25.svg#gh-light-mode-only" width="65%"/>
</p>

<p align=center>
  <img src="https://user-images.githubusercontent.com/16186646/197944452-4a5dcd5f-68d0-4505-b419-e95c298978d7.svg#gh-dark-mode-only" width="65%"/>
</p>

> Besides, `envd` also supports remote build, which means you can build your environment on a remote machine, such as a cloud server, and then push it to the registry. This is especially useful when you are working on a machine with limited resources, or when you expect a build machine with a higher performance.

Reviewer 2: ```suggestion
Besides, `envd` also supports remote build, which means you can build your environment on a remote machine, such as a cloud server, and then push it to the registry. This is especially useful when you are working on a machine with limited resources, or when you expect a build machine with higher performance.
```
